server/liveblog/themes/themes_assets/classic/main.js

```diff
@@ -16,6 +16,11 @@
         function retrieveUpdate() {
             return vm.pagesManager.retrieveUpdate(true);
         }
+        var srcset = '';
+        angular.forEach(config.blog.picture.renditions, function(value) {
+            srcset += ', ' + value.href + ' ' + value.width + 'w';
```

Comment:
Why not using the form 
```
srcset="image.jpg 1x,
             image-320.jpg 320w 1x,
             image-640.jpg 320w 2x"
```
since we have a `max-width` fixed, in this way it would support retina screen